packages/swatch/swatch-group.md

```diff
@@ -144,15 +150,20 @@ The `density` attribute/property is not required and when applied accepts the va
 </sp-swatch-group>
 ```
 
-## Selection
+</sp-tab-panel>
+</sp-tabs>
+
+#### Selection
 
 An `<sp-swatch-group>` element can carry a selection of a `single` swatch or of `multiple` swatches. Then the `selects` property is set to one of these values, the `selected` property will surface an array the represents the string values that have been selected in the UI.
 
 When the value of `selected` is updated via user input, the `change` event will be dispatched on the `<sp-swatch-group>` element to announce that interaction. Calling `preventDefault()` on the `chagne` event will prevent both the `<sp-swatch-group>` and the `<sp-swatch>` that initiated the `change` interaction from updating their `selected` values.
```

Comment:
I have a couple of ideas here that might need some massaging, but I think we clear some things up. 

First, I think maybe we should use the term "selection mode." I've seen that in React, and that makes it a little clearer (at least to me) what exactly we're talking about- that we have a "single selection" mode where a user can only select 1 swatch, while in "multiple selection" mode a user can select more than 1. 

Next, I think we need to clarify that `selected` is both on the `sp-swatch` based on the `change` event, and on the `sp-swatch-group`. I'll leave another comment that sort of rearranges some things in this paragraph to make it a little clearer which things are which.